content/3_Silver/Conclusion.problems.json

```diff
@@ -9,9 +9,7 @@
       "difficulty": "Easy",
       "isStarred": false,
       "tags": ["Prefix Sums"],
-      "solutionMetadata": {
-        "kind": "internal"
-      }
+      "solutionMetadata": { "kind": "internal" }
     },
```

Comment:
could u undo the formatting changes on the file just to keep it consistent with the other one?

thanks 🎉

Reply:
thought precommit should've caught this lowk